core/server/src/main/java/alluxio/master/block/BlockMaster.java

```diff
@@ -157,9 +158,6 @@ public Object getFieldValue(MasterWorkerInfo o) {
   @SuppressFBWarnings("URF_UNREAD_FIELD")
   private Future<?> mLostWorkerDetectionService;
 
-  /** The next worker id to use. This state must be journaled. */
-  private final AtomicLong mNextWorkerId = new AtomicLong(1);
```

Comment:
Well, we never journaled anything. The journaling was never implemented (which is why the bug exists).